app/posts/now-available-in-blue.md

```diff
@@ -0,0 +1,80 @@
+
+Thanks to Sass modules, you can now update the design of a site by calling `govuk-frontend` with the `@use` syntax:
+
+```scss
+$_font-family: system-ui, sans-serif;
+@use "pkg:govuk-frontend/dist/govuk" with (
+  $govuk-font-family: $_font-family,
+  $govuk-brand-colour: #8822aa,
+  $govuk-link-colour: #660088,
+  $govuk-link-hover-colour: #440066,
+  $govuk-link-visited-colour: #333366,
+  $govuk-page-width: 1100px
+);
+```
+
+Generated JavaScript and CSS files are now minimised, and all options for the [Service navigation component](https://design-system.service.gov.uk/components/service-navigation/) are now supported.
+
+And of course, support is provided for the GOV.UK rebrand. Setting the `rebrand` option to `true` will render templates with the refreshed design. This option is set to `false` by default; a later minor update will change this default to `true` before the option is removed.
+
+## GOV.UK Prototype Components
+
+Alongside some under-the-hood updates (moving the project to ES and Sass modules), v4.0 of [this plugin for the GOV.UK Prototype Kit](https://x-govuk.github.io/govuk-prototype-components/) removes the deprecated Primary navigation component. You should use the [Service navigation component](https://design-system.service.gov.uk/components/service-navigation/) to help users navigate your service instead.
+
+## GOV.UK Prototype Filters
+
+Recent updates to the [filters provided by this plugin](https://x-govuk.github.io/govuk-prototype-filters/) for the GOV.UK Prototype Kit have been especially fun.
+
+Turns out JavaScript runtimes have differing opinions around date formatting. How is September abbreviated (‘Sept’ or ‘Sep’)? Should a comma be included in a date that includes a time?
+
+These inconsistencies have been resolved, alongside the ability to include the day of the week in formatted dates and format a person’s age according to the [NHS.UK content guide](https://service-manual.nhs.uk/content/inclusive-content/age).
+
+## GOV.UK Components and Form Builder for Ruby on Rails
+
+[Peter Yates](https://github.com/peteryates) continues his diligent stewardship of the Ruby Gems that provide [components for use with Rails’ ViewComponent framework](https://govuk-components.netlify.app/) and [form components for Rails’ `FormBuilder`](https://govuk-form-builder.netlify.app/).
+
+Updates to both gems in support of the rebrand will be released on 25 June.
+
+## GOV.UK Frontend Statistics
+
+[Frankie Roberto](https://github.com/frankieroberto)’s project attempts to track [adoption of the latest releases to GOV.UK Frontend](https://x-govuk.github.io/govuk-services-frontend-stats/).
+
+Originally created to track adoption of a release that introduced the Tudor crown, the report has been updated to show which services are ready to support the rebrand.
+
+The report isn’t exhaustive (it’s not possible to detect uses of `govuk-frontend` in mono repos or in private repos), but it serves as a good sense check.
```

Comment:
```suggestion
The report isn’t exhaustive (it’s not possible to detect uses of `govuk-frontend` in some types of repository or in closed source projects), but it serves as a good sense check.
```

Reply:
Good suggestion that 😉 (Made the change in one place, but not the place it mattered!)